docs/components/generic/_index.md

```diff
@@ -83,6 +83,13 @@ import (
 )
 ```
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+```cpp
+#include <viam/sdk/services/generic/generic.hpp>
+```
```

Comment:
Out of scope: adding basic include instructions for generic component for CPP as well.